### PR TITLE
Pubx.ai Analytics Adapter: added additional fields to the Auction object

### PR DESCRIPTION
**Pull request by pnhegde**
## Type of change
- [x ] Refactoring (no functional changes, no API changes)

## Description of change
Added additional fields related to pubx, which are stored in the sessionStorage, to the auction object.

### CODE REVIEW

File: modules/pubxaiAnalyticsAdapter.js [hunk 1]
```diff
@@ -21,6 +21,14 @@ let events = {
   deviceDetail: {}
 };
 
+export function getStorage() {
```
Comment (patmmccann) [inline modules/pubxaiAnalyticsAdapter.js:24]: why is this exported?

File: test/spec/modules/pubxaiAnalyticsAdapter_spec.js [hunk 2]
```diff
@@ -28,6 +28,7 @@ describe('pubxai analytics adapter', function() {
     };
 
     let location = utils.getWindowLocation();
+    let storage = window.top['sessionStorage'];
```
Comment (umakajan) [inline test/spec/modules/pubxaiAnalyticsAdapter_spec.js:31]: Are you intending to use [`getStorage()`](https://github.com/prebid/Prebid.js/pull/7723/files#diff-e91c97b69c5deee5b130ed490edff79b671e8f86432cad2705da283f64c9965aR24) here?

Comment (pnhegde) [reply test/spec/modules/pubxaiAnalyticsAdapter_spec.js:31]: Removed the `export` from getStorage() as it might be irrelevant.

**Review by umakajan — APPROVED** [review]

**Comment by pnhegde** [comment]
@patmmccann @umakajan Could you please merge it if everything is fine?

**Pull request merged**